Rakefile

```diff
@@ -18,8 +18,13 @@ end
 ## Development
 
 desc "Run normal tests (exclude no_missing_require_* specs)"
-task :test_normal do
-  sh "bundle exec rspec --exclude-pattern 'spec/line/bot/line_bot_api_gem_spec.rb,spec/line/bot/line_bot_gem_spec.rb'"
+task :test_normal, [:skip_tag] do |_, args|
```

Comment:
skip some tests **only when** special argument is used.
- just running rspec test -> not skipped
- running rspec test with rbs test -> skipped